src/components/responsive-flyout/styles/ResponsiveFlyout.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../../styles/variables';
+@import '../../../styles/constants/media-queries';
+
+.flyout-overlay {
+    .overlay {
+        .overlay-header {
+            display: none;
+        }
+    }
+}
+
+.is-responsive-web {
+    @include breakpoint($medium-screen) {
+        .flyout-overlay.responsive-flyout {
```

Comment:
Does .responsive-flyout duplicate the functionality of .is-responsive-web?

Reply:
No it does not. This is to preserve the non-responsive functionality of ResponsiveFlyout